pyvisa/rname.py

```diff
@@ -5,10 +5,12 @@
 :license: MIT, see LICENSE for more details.
 
 """
+from __future__ import annotations
```

Comment:
Please don't include that future. It is rather unclear if it will ever become the default.

Reply:
Sure thing; I'm pretty sure it's not that necessary at this point. I'll clean up a couple of unrelated changes as well.